### PR TITLE
Update user guide language to be more user-friendly

### LINKED ISSUE #169: Update UG language to be more user friendly

### PR DESCRIPTION
**Pull request by isaactodo**
Closes #169

### CODE REVIEW

**Review by Jianwen0451 — APPROVED** [review]
LGTM!

File: docs/UserGuide.md [hunk 1]
```diff
@@ -5,109 +5,112 @@ title: User Guide
...
 {:toc}
 
 --------------------------------------------------------------------------------------------------------------------
 
 ## Quick start
 
-1. Ensure you have Java `17` or above installed in your Computer.
-   - For Windows 11, see [here](https://www.youtube.com/watch?v=ykAhL1IoQUM)
-   - For MacOS, see [here](https://www.youtube.com/watch?v=lYKHFz8YaD4)
+1. Before you can start using our program, ensure you have Java `17` or above installed in your Computer.
```
Comment (tanyqcollin) [inline docs/UserGuide.md:21]: I think the `start` should be remove.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -231,55 +234,53 @@ Assigns a grade to a module that a student is taking.
...
 Example: `load pa/mybook.json`
 
-The file name must ends with ".json", must not contain any slash "/" and must point to an existing address book .json file.
+The file name must end with ".json", must not contain any slashes "/" and must point to an existing StoreClass .json file.
 
 There should be only one file name provided.
 
 <div markdown="span" class="alert alert-primary">:rotating_light: **Warning:**
-Avoid loading non-address book .json files as it may result in unexpected behaviours
+It is recommended that you avoid loading non-StoreClass .json files, as it may result in unexpected behaviours
```
Comment (tanyqcollin) [inline docs/UserGuide.md:270]: America English of `behaviours` is `behaviors`

**Review by tanyqcollin — APPROVED** [review]
LGTM!

**Review by RadieonAjax — APPROVED** [review]
LGTM!

**Pull request merged**